version2.en.md

```diff
@@ -41,6 +41,7 @@ Moreover, this specification does not contain description of general RTB protoco
     * [native Object](#native-object)
     * [pmp Object](#pmp-object)
     * [deal Object](#deal-object)
+    * [deal extension Object](#dealext-object)
```

Comment:
`s/deal extension/deal.ext/` should make it look more consistent

Reply:
@yowcow 
fixed🙏  https://github.com/voyagegroup/fluct-rtb-spec/pull/47/commits/d84595c472abaad089d98b8d5808b60788a117e8